src/test/java/eu/enmeshed/EnmeshedOnboardingServiceTest.java

```diff
@@ -608,17 +547,15 @@ void itShouldRejectTheIncomingRequestAndReturnSentData() {
     inOrder
         .verify(enmeshedClientMock)
         .searchRelationships(eq(relationshipTemplateId), any(), any());
-    inOrder
-        .verify(enmeshedClientMock)
-        .rejectRelationshipChange(eq(relationshipId), eq(relationshipChangeId), any());
+    inOrder.verify(enmeshedClientMock).rejectRelationship(eq(relationshipId));
     inOrder.verify(enmeshedClientMock).sync();
     inOrder
         .verify(enmeshedClientMock)
         .searchRelationships(eq(relationshipTemplateId), any(), any());
   }
 
   @Test
-  void itShouldPassTheSentAttributesToAcceptanceDecider() {
+  void shouldPassTheSentAttributesToAcceptanceDecider() {
 
     String relationshipTemplateId = "RLT_XXX";
     String relationshipChangeId = "RCH_XXX";
```

Comment:
It is not clear what should I do here

Reply:
Sorry for the confusion, there is nothing more to do in this file than to remove `String relationshipChangeId = "RCH_XXX";` as has already been done (https://github.com/nmshd/connector-java-sdk/pull/4#discussion_r1755022398).

However, regarding the removal of RelationshipChanges, I have also noticed that in the file `RequestResponseSource.java`, the defined `RequestSourceType` should not have a `RelationshipChange` property, but a `RelationshipTemplate` property.

Since no changes have been made to file `RequestResponseSource.java` in this PR so far, I found the comment here to be the most appropriate, as it also dealt with the removal of the RelationshipChanges.

Reply:
No problem, thanks for the explanation! I updated RequestSourceType.